rulesets/phpcs.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<ruleset name="Banovo">
+    <rule ref="../../../escapestudios/symfony2-coding-standard/Symfony/ruleset.xml"/>
+    <rule ref="Generic.Files.LineLength">
+        <properties>
+            <property name="lineLimit" value="120"/>
```

Comment:
our wiki says:
> The hard limit MUST be 120 characters; lines SHOULD be 80 characters or less. (more restrictive than PSR-2)

Do you think we should never warn the user before the 120 chars?